Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM golang:1.15-alpine AS builder
+
+ENV GOFLAGS="-mod=readonly"
+
+RUN apk add --update --no-cache bash ca-certificates curl git gcc g++
+
+RUN mkdir -p /workspace
+WORKDIR /workspace
+
+ARG GOPROXY
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+ARG COMMIT_SHA
+
+COPY . .
+
+RUN set -xe && \
+    export COMMIT_SHA=${COMMIT_SHA:-$(git describe --always --dirty)} && \
+    go build -ldflags "-s -w -X github.com/drakkan/sftpgo/version.commit=${COMMIT_SHA} -X github.com/drakkan/sftpgo/version.date=`date -u +%FT%TZ`" -o sftpgo
+
+
+FROM alpine:3.12
+
+RUN apk add --update --no-cache ca-certificates tzdata bash
+
+SHELL ["/bin/bash", "-c"]
+
+# set up nsswitch.conf for Go's "netgo" implementation
+# https://github.com/gliderlabs/docker-alpine/issues/367#issuecomment-424546457
+RUN test ! -e /etc/nsswitch.conf && echo 'hosts: files dns' > /etc/nsswitch.conf
+
+RUN mkdir -p /data /etc/sftpgo /srv/sftpgo/web /srv/sftpgo/backups
+
+RUN addgroup -g 1000 -S sftpgo
+RUN adduser -u 1000 -h /srv/sftpgo -s /sbin/nologin -G sftpgo -S -D -H sftpgo
+
+VOLUME ["/data", "/srv/sftpgo/backups"]
+
+# Override some configuration details
+ENV SFTPGO_CONFIG_DIR=/etc/sftpgo
+ENV SFTPGO_LOG_FILE_PATH=""
+ENV SFTPGO_HTTPD__TEMPLATES_PATH=/srv/sftpgo/web/templates
+ENV SFTPGO_HTTPD__STATIC_FILES_PATH=/srv/sftpgo/web/static
+
```

Comment:
templates and static files are not supposed to be modified, so we don't need persistent storage for them.

We need persistent storage for SSH host keys, we can set an env var such as this one:

```
SFTPGO_SFTPD__HOST_KEYS=/persistent/path/id_rsa,/persistent/path/id_ecdsa
```

to generate/read them from a persistent path